docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -54,15 +54,47 @@ Some container builds may use parent images from registries that require authent
 . Enter the password for the registry in **Password**.
 . Click **Add secret**.
 
+== Example of creating a quay.io secret
+. Login to Konflux console your workspace (https://console.redhat.com/application-pipeline/workspaces/<your workspace>/applications)
+. Click on `Secrets` on the left menu.
+. Click on `Add Secret`
+. Choose `Image pull secret` for `Secret type`
+. Enter `Secret name` , for example `my-quay-secret` 
+. Choose `Image registry credentials` in `Authentication type` field
+. Enter `quay.io` in `Registry server address`
+. Enter your Quay.io username in `Username`
+. Enter your Quay.io API token in `Password` field.
+. Click on `Add secret`
+. Email is optional
+
+
+Here is the YAML representation of the secret (for reference):
+
+[source,yaml]
+----
+apiVersion: v1
+data:
+.dockerconfigjson: <base64-encoded-credentials>
+kind: Secret
+metadata:
+name: my-quay-secret
+namespace: <your-workspace-tenant>
+type: kubernetes.io/dockerconfigjson
+----
+
 
 [NOTE]
 ====
-Performing this operation through the UI will link the secret to the `appstudio-pipeline`
-`serviceaccount` automatically. If you instead wanted to add manually the secret to the
-namespace (creating the `secret` directly on the tenant
-workspace), you'll need to have the secret linked manually.
+* **Automatic Secret Linking via UI**  
+  The Konflux UI automatically links image pull secrets to the `appstudio-pipeline` ServiceAccount.  
+  - No pipeline YAML changes are required.  
+  - Tasks like the buildah task will use this secret automatically when pulling images from `quay.io`. (https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-oci-ta/0.3/buildah-oci-ta.yaml in volumes) 
+
+* **Manual Secret Creation**  
+  If you create the secret manually (e.g., via `kubectl` or YAML), you must **manually link it** to the `appstudio-pipeline` ServiceAccount in your namespace.
```

Comment:
Do secrets always need to be linked to the ServiceAccount in order for it to be used by a task?

Reply:
another options : 

1. Create secret is in namespace and just have to make sure SA got the right access permissions by Role/RoleBinding 
2. Secret as Env environment 

let me add both ways

Reply:
fixed